src/EnergyPlus/api/func.cc

```diff
@@ -89,7 +89,7 @@ Real64 glycolViscosity(Glycol glycol, Real64 temperature) {
 }
 
 Refrigerant refrigerantNew(const char* refrigerantName) {
-    auto refrigerant = new EnergyPlus::FluidProperties::RefrigerantAPI(refrigerantName);
+    auto *refrigerant = new EnergyPlus::FluidProperties::RefrigerantAPI(refrigerantName);
```

Comment:
@amirroth will love this.  But anyway, Clang Tidy wanted me to use an asterisk here as a way to clarify that I am creating a pointer type object.